pkgs/top-level/all-packages.nix

```diff
@@ -8784,7 +8784,7 @@ with pkgs;
 
   sparse = callPackage ../development/tools/analysis/sparse { };
 
-  speedtest-cli = callPackage ../tools/networking/speedtest-cli { };
+  speedtest-cli = with pythonPackages; toPythonApplication speedtest-cli;
```

Comment:
```suggestion
  speedtest-cli = with python3Packages; toPythonApplication speedtest-cli;
```

Reply:
No, it won't since `python` will probably never refer to `python3`. See https://github.com/NixOS/nixpkgs/issues/18185.

Reply:
okay, no problem